src/BWFMetaEditBundle/Resources/views/Download/raspbianTable.html.twig

```diff
@@ -5,6 +5,29 @@
 <tr>
     <th colspan="5">Notes for installing the packages below:<br>"sudo dpkg -i *deb"<br><br>If you get an error message, use:<br>"sudo apt-get -f install"</th>
 </tr>
+<tr>
+    <th rowspan="4" id="12">12 (Bookworm)</th>
+    <th rowspan="2" id="12.aarch64">aarch64</th>
+    <td><abbr title="Graphical User Interface">GUI</abbr></td>
+    <td><a href="//mediaarea.net/download/binary/bwfmetaedit-gui/25.03/bwfmetaedit-gui_25.03-1_arm64.Raspbian_12.deb">v25.03</a></td>
+    <td>&nbsp;</td>
+</tr>
+<tr>
+    <td><abbr title="Command Line Interface">CLI</abbr></td>
+    <td><a href="//mediaarea.net/download/binary/bwfmetaedit/25.03/bwfmetaedit_25.03-1_arm64.Raspbian_12.deb">v25.03</a></td>
+    <td>&nbsp;</td>
+</tr>
+<tr>
+    <th rowspan="2" id="12.armv7l">armv7l</th>
+    <td><abbr title="Graphical User Interface">GUI</abbr></td>
+    <td><a href="//mediaarea.net/download/binary/bwfmetaedit-gui/25.03/bwfmetaedit-gui_25.03-1_armhf.Raspbian_12.deb">v25.03</a></td>
+    <td>&nbsp;</td>
+</tr>
+<tr>
+    <td><abbr title="Command Line Interface">CLI</abbr></td>
+    <td><a href="//mediaarea.net/download/binary/bwfmetaedit/25.03/bwfmetaedit_25.03-1_armhf.Raspbian_12.deb">v25.03</a></td>
+    <td>&nbsp;</td>
+</tr>
```

Comment:
for  Raspian 11 below, not updated (and not "old", not coherent) but [bwfmetaedit-gui_25.03-1_arm64.Raspbian_11.deb](https://mediaarea.net/download/binary/bwfmetaedit-gui/25.03/bwfmetaedit-gui_25.03-1_arm64.Raspbian_11.deb) (all 4 lines).